draft-quic-atsss-reqs.mkd

```diff
@@ -74,49 +77,48 @@ as shown here.
 
 [comment]: # (OB: not sure we need that section, we'll see later)
 
+# Terminology 
+
+This document makes use of 3GPP-specific terms defined in RFC6459, mainly the following ones: 
+
+-    Packet Data Network (PDN): is a packet-based network that
+      either belongs to the operator or is an external network such as
+      the Internet or a corporate intranet.  The user eventually
+      accesses services in one or more PDNs.  The operator's packet core
+      networks are separated from packet data networks either by GGSNs, PDN Gateways (PDN-GWs), or User Plane Function (UPF).
+
+-     UE (User Equipment): refers to the devices that are hosts with the
+      ability to obtain Internet connectivity via a 3GPP network.  A MS
+      is comprised of the Terminal Equipment (TE) and a Mobile Terminal
+      (MT).  
```

Comment:
The terminologies MS, TE and MT seem GSM/2G specific.

Reply:
That was to cover tethering and dongle cases, but this is fair. 

Fixed now in 54ecab3. Thank you.